contracts/pcv/PCVGuardian.sol

```diff
@@ -117,7 +120,10 @@ contract PCVGuardian is IPCVGuardian, CoreRef {
     {
         // improbable to ever overflow
         unchecked {
-            require(_whitelistAddresses.length != 0, "empty");
+            require(
+                _whitelistAddresses.length != 0,
```

Comment:
I don't think we need this check as the function will do nothing if the length is 0